SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialTraceUniqueId.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/Connection/SpatialTraceUniqueId.h"
+
+#include "Utils/SchemaUtils.h"
+
+using namespace SpatialGDK;
+
+namespace
+{
+void ToHex(uint8 N, TCHAR OutHex[3])
+{
+	const TCHAR* Chars = TEXT("0123456789ABCDEF");
+	OutHex[0] = Chars[(N >> 4) & 0xF];
+	OutHex[1] = Chars[N & 0xF];
+#if !PLATFORM_LITTLE_ENDIAN
+	std::swap(OutHex[0], OutHex[1]);
+#endif
+	OutHex[2] = '\0';
+}
+} // namespace
+
+FString EventTraceUniqueId::GetString() const
+{
+	FString Str;
+	TCHAR Tmp[3];
+	for (int i = 0; i < sizeof(internal_bytes); i++)
```

Comment:
Possible nit: not sure what our standard is but I wonder if we generally avoid "int" and use int32 by default?